s30020-NikiforKoseda/zadanie 1.txt

```diff
@@ -0,0 +1,90 @@
+1.Uruchom terminal.
+Konfiguracja PUTTY, logowanie na szuflandię i wszystko elegancko działa
+
+2.Wyświetl zawartość aktualnego katalogu.
+ls wpisane
+
+3.Sprawdź gdzie aktualnie się znajdujesz.
+pwd wpisane
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej
+cd // wpisane
+
+5.Sprawdź gdzie jesteś.
+pwd ponownie wpisane
+
+6.Wyświetl zawartość bieżącego katalogu.
+ls ponownie wpisane
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z
+ odpowiednim parametrem).
+ls $home wpisane
+
+8.Przejdź do katalogu Obrazy (lub innego katalogu, który jest dostępny - podaj 
+wprost, że zmieniasz katalog na inny bo np. Obrazy nie są u Ciebie dostępne) w 
+twoim katalogu domowym za pomocą ścieżki względnej.
+cd media wpisane
+
+9,Sprawdź gdzie jesteś.
+pwd ponownie wpisane
+
+10.Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././
+W opisie rozwiązania proszę umieścić również to jak rozumiesz co się właściwie 
+stało. Dlaczego znalazła/eś się w tym miejscu do którego przeniosła Cię ta ście
+żka? Czy dodanie kilku ./././././ na początku/środku/końcu powoduje, że ścieżka
+się zmieni? Co ta ścieżka oznacza/sobą reprezentuje?
+
+zostaję w tym samym folderze, ./ to odnosik do aktualnego folderu. Dodani-
+e większej ilości nic nie zmieni, nawzajem się kasują. Ta ścieżka zasadniczo
+wskazuje na nasz aktualny folder
+
+11.Przejdź do swojego katalogu domowego za pomocą ścieżki bezwzględnej.
+ls /home/PJWSTK/s30020 wpisane
+
+12.Zobacz co się stanie jak wpiszesz komendę 'cd /root'. Jak już to zrobisz, za
+stanów się co się stało. Poszukaj w Internecie.
+Odrzuciło mi dostęp do folderu administratora, albowiem nie mam do niego d-
+ostępu
+
+13.Zobacz, co się stanie, jeśli wciśniesz kombinację klawiszy: Ctrl+Alt+F3
+Sprawdź w Internecie co się dzieje kiedy używa się takiej kombinacji. W Interne
+cie mogą pojawić się wersje z F1-F6. To de facto to samo. Odpowiedź "wylogowało
+ mnie" jest błędna. Nie. Nie wylogowało. Szukaj dalej.
+Przełączy mnie na terminal tekstowy
+
+14.A teraz co się stanie po: Alt+F1 lub ALT+F2 lub ALT+F7 (na starszych linuxac
+h)
+Przełącza między terminalami tekstowymi
+
+
+1.Korzystając z komend cd oraz mkdir stwórz następujące drzewo katalogów w swo
+im katalogu domowym (w dowolnym podkatalogu, albo bezpośrednio w ~): 
+Kilka mkdirów i elegancko ogarnięte
```

Comment:
Nie podoba mi się taka odpowiedź. Brakuje tu części techniczej.
Sama luźna forma wypowiedzi może pozostać ale bez podania co konkretnie zostało wykonane oraz co uzyskałeś jak również jakie wnioski z tego płyną, to zadanie jest bezcelowe wtedy.